crates/wallet/src/test_utils.rs

```diff
@@ -160,6 +160,13 @@ pub fn get_test_single_sig_cltv() -> &'static str {
     "wsh(and_v(v:pk(cVpPVruEDdmutPzisEsYvtST1usBR3ntr8pXSyt6D2YYqXRyPcFW),after(100000)))"
 }
 
+/// `wsh` descriptor with policy `and(pk(A),after(500_000_000))`
+/// the parameter passed to miniscript fragment `after` has to equal or grather than 500_000_000
+/// in order to use a lock based on unix time
+pub fn get_test_single_sig_cltv_for_specific_time() -> &'static str {
+    "wsh(and_v(v:pk(cVpPVruEDdmutPzisEsYvtST1usBR3ntr8pXSyt6D2YYqXRyPcFW),after(500000000)))"
```

Comment:
⛏️  just a nit but we should use a timestamp that's sometime after the genesis block. Sometimes people use these tests as examples so will make more sense to use a more recent time. Also the number is easier to read in the code with the underscores.
```suggestion
    "wsh(and_v(v:pk(cVpPVruEDdmutPzisEsYvtST1usBR3ntr8pXSyt6D2YYqXRyPcFW),after(1_734_230_218)))"
```

Reply:
I tried to update the descriptor as you mentioned, but there is an error with the format:
```descriptors must be valid: Miniscript(Unexpected("unexpected «1_734_230_218»"))```
It looks like miniscript does not have support for numbers with underscores.